binding.gyp

```diff
@@ -138,6 +139,15 @@
           ],
           'conditions': [
             ['OS=="win"', {
+              'copies': [{
+                'destination': '<(PRODUCT_DIR)',
+                'files': [
+                  '<(jpeg_root)/bin/jpeg62.dll',
+                ]
+              }],
+              'include_dirs': [
+                '<(jpeg_root)/include'
+              ],
               'libraries': [
                 '-l<(GTK_Root)/lib/jpeg.lib'
```

Comment:
Maybe I've botched my GTK dir, but I don't have a (GTK_Root)/lib/jpeg.lib, so I get a linker error. Did you add this file manually?

#458 changes it to `'-l<(jpeg_root)/lib/jpeg-static.lib'`, but that does not build.

If I change it to `'-l<(jpeg_root)/lib/jpeg.lib'`, it builds, but see comment in main discussion.